staging/src/k8s.io/apiserver/pkg/util/flowcontrol/context.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"context"
+	"net/http"
+
+	apitypes "k8s.io/apimachinery/pkg/types"
+)
+
+type priorityAndFairnessKeyType int
+
+const priorityAndFairnessKey priorityAndFairnessKeyType = iota
+
+const (
+	matchedFlowSchemaUIDHeader    = "FlowSchema-UID"
```

Comment:
Since these are non-standard HTTP headers, should they start with "X-"?

Reply:
```
I0217 16:03:23.979268   67292 round_trippers.go:444] Response Headers:
I0217 16:03:23.979279   67292 round_trippers.go:447]     Content-Length: 132
I0217 16:03:23.979285   67292 round_trippers.go:447]     Date: Mon, 17 Feb 2020 08:03:23 GMT
I0217 16:03:23.979293   67292 round_trippers.go:447]     Audit-Id: b7a405c4-decc-4786-91f8-0ce426626a27
```
i followed the existing naming style of `Audit-Id` from response headers. and also i remember there's a RFC proposing deprecating support for "X-*" headers..

Reply:
What is the RFC?  I wonder what is its status.

Reply:
https://tools.ietf.org/html/rfc6648 it's a "BEST CURRENT PRACTICE"